python/lsst/pipe/tasks/calibrate.py

```diff
@@ -25,88 +25,117 @@
-        target = DetectAndMeasureTask,
-        doc = "Detect sources to high sigma, deblend and peform single-frame measurement",
+        dtype=bool,
+        default=True,
+        doc="Write reference matches (ignored if doWrite false)?",
     )
     doAstrometry = pexConfig.Field(
-        dtype = bool,
-        default = True,
-        doc = "Perform astrometric calibration?",
+        dtype=bool,
+        default=True,
+        doc="Perform astrometric calibration?",
     )
     astrometry = pexConfig.ConfigurableField(
-        target = AstrometryTask,
-        doc = "Perform astrometric calibration to refine the WCS",
+        target=AstrometryTask,
+        doc="Perform astrometric calibration to refine the WCS",
     )
     requireAstrometry = pexConfig.Field(
-        dtype = bool,
-        default = True,
-        doc = "Raise an exception if astrometry fails? Ignored if doAstrometry false.",
+        dtype=bool,
+        default=True,
+        doc="Raise an exception if astrometry fails? Ignored if doAstrometry false.",
     )
     doPhotoCal = pexConfig.Field(
-        dtype = bool,
-        default = True,
-        doc = "Perform phometric calibration?",
+        dtype=bool,
+        default=True,
+        doc="Perform phometric calibration?",
     )
     requirePhotoCal = pexConfig.Field(
-        dtype = bool,
-        default = True,
-        doc = "Raise an exception if photoCal fails? Ignored if doPhotoCal false.",
+        dtype=bool,
+        default=True,
+        doc="Raise an exception if photoCal fails? Ignored if doPhotoCal false.",
     )
     photoCal = pexConfig.ConfigurableField(
-        target = PhotoCalTask,
-        doc = "Perform photometric calibration",
+        target=PhotoCalTask,
+        doc="Perform photometric calibration",
     )
     icSourceFieldsToCopy = pexConfig.ListField(
-        dtype = str,
-        default = ("calib_psfCandidate", "calib_psfUsed", "calib_psfReserved"),
-        doc = "Fields to copy from the icSource catalog to the output catalog for matching sources "
+        dtype=str,
+        default=("calib_psfCandidate", "calib_psfUsed", "calib_psfReserved"),
+        doc="Fields to copy from the icSource catalog to the output catalog for matching sources "
             "Any missing fields will trigger a RuntimeError exception. "
-            "If detectAndMeasure.doMeasureApCorr is True and detectAndMeasure cannot determine its own "
-            "suitable candidates, then this list must include "
-            "config.detectAndMeasure.measureApCorr.inputFilterFlag. "
             "Ignored if icSourceCat is not provided."
     )
     matchRadiusPix = pexConfig.Field(
-        dtype = float,
-        default = 3,
-        doc = "Match radius for matching icSourceCat objects to sourceCat objects (pixels)",
+        dtype=float,
+        default=3,
+        doc="Match radius for matching icSourceCat objects to sourceCat objects (pixels)",
```

Comment:
Please don't include the above changes removing the spaces around the `=` unless it's in a separate commit.